tests/test_a_froidure_pin.py

```diff
@@ -246,7 +246,7 @@ def test_factorisation_and_relations(self):
             S.word_to_element([0, 1, 0, 1]), Transformation([0, 3, 4, 1, 2])
         )
         self.assertEqual(S.word_to_pos([0, 1, 0, 1]), 15)
-        
+
```

Comment:
I didn't edit this file myself, so I imagine these are the formatter's changes

Reply:
Looks like it